Plugin/test1/src/main/java/util/InstrumentationResultMessage.java

```diff
@@ -0,0 +1,215 @@
+        builder.append("Class: ").append(psiClass.getQualifiedName()).append("\n");
+        return this;
+    }
+
+    /**
+     * Append the class or interface qualified name to the result message
+     *
+     * @param psiMethod A Java method or constructor.
+     * @return A instance of this object
+     */
+    @SuppressWarnings("UnusedReturnValue")
+    public InstrumentationResultMessage appendPsiMethod(@NotNull PsiMethod psiMethod) {
+        builder.append("Method: ").append(psiMethod.getName()).append("\n");
+        return this;
+    }
+
+    /**
+     * Append a string to inform the class initializer was instrumented to the result message
+     *
+     * @return A instance of this object
+     */
+    @SuppressWarnings("UnusedReturnValue")
+    public InstrumentationResultMessage appendPsiClassInitializer() {
+        builder.append("Initializer block: ").append("\n");
+        return this;
+    }
+
+    /**
+     * Append the field name to the result message
+     *
+     * @param psiField A Java field or enum constant.
+     * @return A instance of this object
+     */
+    @SuppressWarnings("UnusedReturnValue")
+    public InstrumentationResultMessage appendPsiField(@NotNull PsiField psiField) {
+        builder.append("Field: ").append(psiField.getName()).append("\n");
+        return this;
+    }
+
+    /**
+     * Append the statement text to the result message
+     *
+     * @param psiStatement A Java statement.
+     * @return A instance of this object
+     */
+    public InstrumentationResultMessage appendPsiStatement(@NotNull PsiStatement psiStatement) {
+        builder.append("Statement: ").append(psiStatement.getText()).append("\n");
+        return this;
+    }
+
+    /**
+     * Append a block separator to the result message
+     *
+     * @return A instance of this object
+     */
+    @SuppressWarnings("UnusedReturnValue")
+    public InstrumentationResultMessage appendNewBlock() {
+        builder.append("\n");
+        return this;
+    }
+
+    /**
+     * Takes the constructed message in {@code builder} and make a dialog with the result
+     *
+     * @param project An object representing an IntelliJ project.
+     * @param title   The title of the dialog
+     */
+    public void showResultDialog(Project project, String title) {
+        Messages.showMessageDialog(project, getMessage(), title, Messages.getInformationIcon());
+    }
+
+    /**
+     * Make a error dialog with the {@code exception} content
+     *
+     * @param project   An object representing an IntelliJ project.
+     * @param exception The caught exception
+     * @param title     The title of the dialog
+     */
+    public void showErrorDialog(Project project, @NotNull Exception exception, String title) {
+        StringWriter errors = new StringWriter();
+        exception.printStackTrace(new PrintWriter(errors));
+        Messages.showErrorDialog(project, errors.toString(), title);
+    }
```

Comment:
I created the dialog methods mainly because of the error dialog, to avoid the need to extract the exception message in all actions.

I was just wondering if these should be in a third utility class.

If you agree it would be better I will move them.